blockstore/apps/search/documents.py

```diff
@@ -0,0 +1,28 @@
+"""
+Documents for search app.
+"""
+
+from .core.documents import Document
+from .fieldsets import (
+    AnalyticsFieldSet,
+    AuthorshipFieldSet,
+    EntityFieldSet,
+    OwnernershipFieldSet,
+    SummaryFieldSet,
+    TagsFieldSet,
+)
+
+
+class BlockDocument(Document):
+    """
+    Document for entities to be indexed in BlocksIndex.
+    """
+
+    FIELD_SETS = (
```

Comment:
We can figure out how to make this configurable but there are too many unknowns right now so going with hardcoding this list for now.